jsapp/js/account/plan.module.scss

```diff
@@ -0,0 +1,232 @@
+@use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
+@use 'scss/breakpoints';
+
+.accountPlan {
+  padding: sizes.$x30 sizes.$x40;
+  overflow-y: auto;
+  position: relative;
+  height: 100%;
+  width: 100%;
+}
+
+.plansSection {
+  display: grid;
+  grid-template-rows: repeat(4, auto);
+  grid-template-columns: repeat(auto, 1fr);
+  column-gap: sizes.$x20;
+}
+
+.intervalToggle {
+  grid-column: 1 / 5;
+  grid-row: 1/2;
+  justify-self: center;
+  background-color: colors.$kobo-cloud;
+  padding: sizes.$x10 sizes.$x4;
+  border-radius: sizes.$x4;
+}
+
+.intervalToggle input[type="radio"] {
+  display: none;
+  position: absolute;
+  width: 100%;
+  appearance: none;
+}
+
+.intervalToggle label {
+  padding: sizes.$x8 sizes.$x12;
+  color: colors.$kobo-dark-blue;
+}
+
+.intervalToggle input[type="radio"]:checked+label {
+  background-color: colors.$kobo-blue;
+  animation-name: blink;
+  animation-duration: 1s;
+  color: colors.$kobo-white;
+  padding: sizes.$x8 sizes.$x12;
+  border-radius: sizes.$x4;
+}
+
+.intervalToggle input[type="radio"]:checked+label:after {
+  background-color: colors.$kobo-blue;
+  padding: sizes.$x8 sizes.$x12;
+}
+
+.intervalToggle input[type="radio"]:checked+label:before {
+  width: 20px;
+  height: 20px;
+  color: colors.$kobo-cloud;
+  padding: sizes.$x8 sizes.$x12;
+}
+
+.planContainer {
+  grid-auto-columns: 1fr;
+  width: 25em;
+  grid-row: 3/4;
+  border-radius: sizes.$x6;
+  border: colors.$kobo-gray-85 solid 1px;
+  padding: 27px 54px;
+}
+
+.priceName,
+.enterpriseTitle {
+  color: colors.$kobo-gray-40;
+  font-weight: 700;
+  font-size: sizes.$x24;
+}
+
+.priceName {
+  text-align: center;
+}
+
+.enterpriseTitle {
+  margin: 0 0 sizes.$x16;
+
+}
+
+.priceTitle {
+  color: colors.$kobo-dark-blue;
+  text-align: center;
+  font-size: sizes.$x18;
+  padding-bottom: sizes.$x16;
+  font-weight: 600;
+}
+
+.line {
+  border-bottom: 1px solid colors.$kobo-gray-85;
+  border-radius: 5px;
+  width: 100%;
+  margin-bottom: 20px;
```

Comment:
Also as noted in other comment - it would be simpler if only the line would have top and bottom margins, and both button and the list below wouldn't

Reply:
If there is something that does have a variable size should I add it? What are our rules for adding sizes?

Reply:
The rule for missing size variable is something like this:
- if the size seems weird and unusual (e.g. `23px`, `17px` etc.) and we have something almost identical we should use the existing one (e.g. `$x18` instead of `17px`)
- if the size seems something common (e.g. `32px`, repetitions of 10 or 5, etc.) we should probably add it, unless there is something similar already that works well in given situation (e.g. Figma uses `25px`, but we have already `$x24` and the difference in the visible UI is minimal - we should go with existing 24)
- in other situation just add the size and use it :)

Reply:
Sounds good!